src/ZoneServer/Skills/Handlers/Highlander/CrossCut.cs

```diff
@@ -0,0 +1,98 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Melia.Shared.Data.Database;
+using Melia.Shared.L10N;
+using Melia.Shared.Tos.Const;
+using Melia.Shared.World;
+using Melia.Zone.Network;
+using Melia.Zone.Skills.Combat;
+using Melia.Zone.Skills.Handlers.Base;
+using Melia.Zone.Skills.SplashAreas;
+using Melia.Zone.World.Actors;
+using Melia.Zone.World.Actors.CombatEntities.Components;
+using static Melia.Zone.Skills.SkillUseFunctions;
+
+namespace Melia.Zone.Skills.Handlers.Highlander
+{
+	/// <summary>
+	/// Handler for the Highlander skill Crosscut.
+	/// </summary>
+	[SkillHandler(SkillId.Highlander_CrossCut)]
+	public class CrossCut : IGroundSkillHandler
+	{
+		/// <summary>
+		/// Handles skill, damaging targets.
+		/// </summary>
+		/// <param name="skill"></param>
+		/// <param name="caster"></param>
+		/// <param name="originPos"></param>
+		/// <param name="farPos"></param>
+		public void Handle(Skill skill, ICombatEntity caster, Position originPos, Position farPos, ICombatEntity target)
+		{
+			if (!caster.TrySpendSp(skill))
+			{
+				caster.ServerMessage(Localization.Get("Not enough SP."));
+				return;
+			}
+
+			skill.IncreaseOverheat();
+			caster.SetAttackState(true);
+
+			var splashParam = skill.GetSplashParameters(caster, originPos, farPos, length: 50, width: 100, angle: 90);
+			var splashArea = skill.GetSplashArea(SplashType.Fan, splashParam); // DB states that this should be Square
```

Comment:
I was mainly using fan because from my initial testing, Square splash areas were working strangely, but after fiddling with them a bit more I think I got it working reasonably, so I've switched it to Square as the skill database indicates.  It's worth noting though that the size of the area is not specified at all in skill-db, so I've tweaked the values as best I can to match the animation.

Reply:
Our current understanding is that skill_bytool is the data set we want to reference for skill hit areas, as the data has been reliable several times already. According to the data for Cross Cut, the length should be 30 and the width 45. Note that 45 is half the width, which extends in both directions for a total of 90 internally. Do you think these parameters could work?

Reply:
I've switched it to the new parameters, they seem close enough.